src/routes/UnsyncedRecordsRoute.tsx

```diff
@@ -0,0 +1,854 @@
+			truckInYardHistory.targetWeight &&
+				Object.assign(record.details, { "Target Weight": truckInYardHistory.targetWeight });
+			truckInYardHistory.request?.requestKey &&
+				Object.assign(record.details, { Request: truckInYardHistory.request.info });
+			if (truckInYardHistory.orderProduct?.orderProductKey) {
+				Object.assign(record.details, {
+					"In-Yard Customer":
+						truckInYardHistory.orderProduct?.order.customer.customerID +
+						" - " +
+						truckInYardHistory.orderProduct?.order.customer.customerName,
+				});
+				Object.assign(record.details, {
+					"In-Yard Order":
+						truckInYardHistory.orderProduct?.order.orderNumber +
+						" - " +
+						truckInYardHistory.orderProduct?.order.description,
+				});
+				Object.assign(record.details, {
+					"In-Yard Product":
+						truckInYardHistory.orderProduct?.product.productID +
+						" - " +
+						truckInYardHistory.orderProduct?.product.productDescription,
+				});
+			}
+
+			truckInYardHistory.request?.requestKey &&
+				Object.assign(record.details, { "Request Key": truckInYardHistory.request.requestKey });
+			if (truckInYardHistory.orderProduct?.orderProductKey) {
+				Object.assign(record.details, {
+					"In-Yard Customer Key": truckInYardHistory.orderProduct?.order.customer.customerKey,
+				});
+				Object.assign(record.details, {
+					"In-Yard Order Key": truckInYardHistory.orderProduct?.order.orderKey,
+				});
+				Object.assign(record.details, {
+					"In-Yard OrderProduct Key": truckInYardHistory.orderProduct?.orderProductKey,
+				});
+				Object.assign(record.details, {
+					"In-Yard Product Key": truckInYardHistory.orderProduct?.product.productKey,
+				});
+			}
+
+			unsyncedRecords.push(record);
+		});
+
+		return unsyncedRecords.sort((a, b) => compareAsc(new Date(a.date), new Date(b.date)));
+	}, [unsyncedTicketsQuery, unsyncedTrucksQuery, unsyncedTrucksInYardQuery, unsyncedTruckInYardHistoryQuery]);
+
+	const numberOfErrors = unsyncedRecordsData
+		.map((r) => {
+			if (r.syncedError != "" && r.syncedError != null) {
+				return r.syncedError;
+			}
+			return;
+		})
+		.filter((r) => r).length;
+
+	// Building the grid
+	const gridRef = useRef<AgGridReact>(null);
+	const [quickSearch, setQuickSearch] = useState("");
+
+	const gridOptions: GridOptions = {
+		columnDefs: [
+			{
+				field: "type",
+				headerName: "Type",
+				minWidth: 90,
+			},
+			{
+				field: "date",
+				headerName: "Date",
+				minWidth: 90,
+				valueFormatter: (params) => {
+					return utcToLocalDatetime(params.value.toString(), "MM/dd/yy hh:mm a");
+				},
+			},
+			{
+				field: "details",
+				headerName: "Record ID",
+				minWidth: 120,
+				valueFormatter: (params) => {
+					const data = Object.entries(params.value);
+					const record = data
+						.find((d) => {
+							const dLower = d[0].toLowerCase();
+							return dLower == "ticket #" || dLower == "truck id";
```

Comment:
Should use === in almost all cases
```suggestion
							return dLower === "ticket #" || dLower === "truck id";
```